src/App.js

```diff
@@ -44,7 +49,8 @@ class App extends Component {
         <Navbar />
         {
           this.state.error && 
-          <h2>{this.state.error} </h2> 
+          <ErrorPage 
```

Comment:
That's a nice way to make an error by creating a separate component! I didnt think about that